src/open_inwoner/templates/pages/contactmoment/list.html

```diff
@@ -39,4 +63,31 @@ <h1 class="utrecht-heading-1" id="contactmomenten">{{ page_title }} ({{ contactm
     {% if contactmomenten %}
         {% pagination page_obj=page_obj paginator=paginator request=request %}
     {% endif %}
+
+    <section class="contactmomenten contactmomenten__contact-form">
+        <div class="form-container">
+            <div class="form-heading">
+                <h2 class="h2">{% trans "Stel een vraag" %}</h2>
+                <p class="p">
+                {% blocktrans trimmed %}
+                    Heeft u een vraag? Dan kunt u deze hier stellen.
+                    Een van onze medewerkers beantwoord uw vraag z.s.m.
+                {% endblocktrans %}
+                </p>
+            </div>
+            <div>
+                <form action="#" method="POST" id="question-form">
+                    {% csrf_token %}
+                    {% for field in question_form.fields %}
+                        {% autorender_field question_form field %}
+                    {% endfor %}
+
+                    <button class="button button--textless button--icon button--icon-after button--primary" type="submit">
```

Comment:
We do not like to float :-) and it's missing an icon - Here it's easier to use the HTML that already exists in '_src/open_inwoner/templates/pages/cases/contact_form.html_' and replace this button with this (containing the icon too):
 
```HTML
<div class="form__submit">
  {% button_row %}
   {% button text=_("Vraag versturen") icon="east" icon_position="after" icon_outlined=True id="submit_contact" type="submit" disabled=True primary=True name="contact_form_submit" %}
  {% endbutton_row %}
</div>
```

Also it's missing the disabled button behaviour - for example look at '_src/open_inwoner/js/components/cases/case_contact_form.js_' so you can add a new JS file for this Vragen Form submit button,  - or make 1 file that looks for a new form-selector that can be enabled or disabled depending on whether the user is actually typing something, in such a way the JS can be used by both forms so it becomes a more general JS styling thing.